shesha-reactjs/src/providers/form/models.ts

```diff
@@ -189,6 +190,8 @@ export interface IConfigurableFormComponent
   noDataSecondaryText?: string;
 
   permissions?: string[];
+
+  allowClear?: boolean;
```

Comment:
Why is this property here? It is clearly specific to some component (I assume ColorPicker). What exactly in ALL components should be allowed to be cleared?

Reply:
I noticed that it was supported in a couple of input components e.g. Input, datePicker, colorPicker, autocomplete, and Select, so I thought moving it up was a good idea.  Now I see it's not because very few components support it.